crates/nostr/src/event/tag/standard.rs

```diff
@@ -1245,6 +1248,22 @@ where
     })
 }
 
+fn parse_t_tag<S>(tag: &[S]) -> Result<TagStandard, Error>
+where
+    S: AsRef<str>,
+{
+    let hashtag = tag.get(1).ok_or(Error::UnknownStandardizedTag)?.as_ref();
+
+    // Not all languages have uppercase and lowercase versions of letters.
+    // If not all letters are lowercase and there is no single uppercase letter,
+    // it means the hashtag is in a language that doesn't have uppercase/lowercase letters.
+    if !hashtag.chars().all(char::is_lowercase) && hashtag.chars().any(char::is_uppercase) {
```

Comment:
Can we consider a hashtag valid if the `char::is_uppercase` returns `false` for all chars?

Reply:
Yes, so I should keep `hashtag.chars().any(char::is_uppercase)` also it's faster, because it will returns `true` on the first uppercase letter.